src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -193,7 +193,7 @@ namespace {
 
     bool is_rate(const std::string& keyword) {
         static const keyword_set ratekw {
-            "OPR", "GPR", "WPR", "LPR", "NPR", "VPR",
+            "OPR", "GPR", "WPR", "WGLIR", "LPR", "NPR", "VPR",
```

Comment:
This needs to be `"GLIR"`.  We [strip off](https://github.com/OPM/opm-common/blob/6ee4e5ce9a8ca272bb91281ee7e28bef6b305998/src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp#L210) the leading letter before we look up keywords in this table.